string topic_name
int32 motor_a_speed 
float32 motor_a_rotation
int32 motor_b_speed
float32 motor_b_rotation